README.md

```diff
@@ -96,11 +96,11 @@ Generally you can [perform a rollback](https://docs.fedoraproject.org/en-US/fedo
 
 To rebase onto a specific date, use a date tag:
 
-    rpm-ostree rebase ostree-unverified-registry:ghcr.io/ublue-os/silverblue-nvidia:20230128
+    rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/silverblue-nvidia:20230128
 
 Or to rebase onto a specific release, driver, and date:
 
-    rpm-ostree rebase ostree-unverified-registry:ghcr.io/ublue-os/silverblue-nvidia:37-525-20230128
+    rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/silverblue-nvidia:37-525-20230128
```

Comment:
For these two lines (99 and 103) since they use date-tagged images, the dates should probably be updated to a version which includes/supports the new policy files for signed images.